libs/components/indicators/src/lib/modules/alert/alert.modern.component.scss

```diff
@@ -27,16 +27,16 @@
         --sky-override-alert-text-vertical-padding,
         var(--sky-space-inset-balanced-none)
       )
-      var(--sky-space-text_action_gap-m)
+      var(--sky-space-gap-text_action-m)
       var(
         --sky-override-alert-text-vertical-padding,
         var(--sky-space-inset-balanced-none)
       )
-      var(--sky-space-icon_gap-l);
+      var(--sky-space-gap-icon-l);
     width: 100%;
 
     ::ng-deep a {
-      color: change-color($sky-text-color-default, $alpha: 0.8);
+      color: hsl(from var(--sky-text-color-default) h s l / 0.8);
```

Comment:
I discovered a different manner of doing this calculation which allows us to use the token. I did it in toast but also updated it here.